### DISCUSSION TITLE
.zip files not loading, no Tools icon in left nav

### DISCUSSION THREAD

**Discussion by damiano1976** [pimcore/pimcore#10713] [Q&A / Support]
Version: 
10.2

Issue: 
When loading assets to a fresh install of Pimcore, I cannot open .zip files that have been added. The file will show a "spinning" icon in the left nav, but never open.

Also, the profiler shows multiple GET statements being generated seemingly without end.
Example from one of the API calls: ~/admin/asset/get-data-by-id?_dc=1635349557290&id=414&type=unknown

Tried to clear the browser cache, different browser...no luck.

As a side-note, I seem to be missing my "Tools" button to check the DB. Any help with either of these issues would be appreciated. Thanks!

![Inkedpimcore-issue-1_LI](https://user-images.githubusercontent.com/17390779/139100312-1f6a9e59-9891-499e-ba7d-6a7ac2769c45.jpg)

**Comment by dvesh3** [comment]
@damiano1976 Thanks for reporting the zip file issue. I can reproduce the bug and reported here https://github.com/pimcore/pimcore/issues/10716

Regarding no Tools icon in left nav, please check if you are on default perspective here:
![image](https://user-images.githubusercontent.com/5137917/139208310-73facf42-bb49-45d4-b7bf-abea7f436f92.png)

**Reply by damiano1976** [reply]
@dvesh3 Thank you for the quick reply! You were correct with the Perspective...looks like it was changed somehow. All good there.

As for the zip issue, I appreciate the support ticket but do we know if this issue is on previous versions of Pimcore? If an older version does not contain this bug, I could use that as a work-around until 10.2 can be patched/addressed.

Thank you again for you help! Much appreciated!

**Reply by dvesh3** [reply]
@damiano1976 This regression introduced in 10.0.8, so you could use any version lower than that.
